ownership.md

```diff
@@ -0,0 +1,56 @@
+    - Network-layer restrictions may also be recommended based on the context
+- All new 3rd party software / vendors used as part of a service should have approval via the [official process](https://spike.gnmremote.com/content/178/requesting-software-for-your-gnm)
+
+### Infrastructure
+- AWS infrastructure should *not* be deployed to the Developer Playground account
+- All new AWS infrastructure should be defined using [CDK](https://github.com/guardian/cdk), and legacy services should be migrated when possible
+- Infrastructure costs should be estimated and within existing budget (unless otherwise approved by Head/Director of Engineering)
+
+### Observability and support
+- Monitoring and alerting should exist to ensure the owner is notified when the core functionality of a service is unavailable or impaired
+- Application logs and (if necessary/applicable) telemetry data should be [shipped](https://github.com/guardian/deploy-tools-platform/tree/main/elk), so that service impairment can be debugged effectively for common problems (e.g. application errors, resource constraints)
+- For high priority applications, a runbook should be created to describe to other engineers how to debug and address service impairment incidents ([example](https://docs.google.com/document/d/1Mz0cp0Ktq1IaOoVd-kqWF_VK25g9E6QQXiiXCTqbXno/edit#heading=h.y64rwxfd5dwk))
+
+## High level maintenance expectations
+
+The broad expectations of ongoing maintenance of a production application are:
+1. Security vulnerabilities are [addressed](/security.md#vulnerability-management) *as a priority*
+1. Dependencies are [kept up to date](https://github.com/guardian/security-hq/blob/73d5174a49efcfda009c804305d748edced6a8af/hq/markdown/vulnerability-management.md#vulnerabilities-via-3rd-party-libraries), so that security patching requirements can be met efficiently
+1. Unused functionality is removed
+1. Impaired functionality is addressed, with priority determined by its importance
+1. Costs are monitored. [Baseline monitoring](https://github.com/guardian/aws-cost-management) is added globally
+1. Architecture/design is reviewed periodically to ensure adherence to best practices and any SLA/SLOs at the team or org level
+
+### Orphaned Projects
+In the vast majority of cases we address these by assigning [ownership](./github.md#collaborators-and-codeowners) of source code to an official engineering team. Official engineering teams have useful properties for this task, like resourcing oversight and planning ceremonies.
+
+However there are some repositories (containing production software) we have created that have no obvious official engineering team owner. Instead we expect a small group of volunteers to be responsible for the above maintenance tasks. We expect these volunteers to:
+- Create a GitHub team and add volunteers as team members. Notify the Developer Experience team of this (the team should be admins of all relevant GitHub repos)
```

Comment:
We suggest an EM be involved in this process. As EMs have GH admin rights, can we drop the need to notify DevX here?

Reply:
Unfortunately we still have a need to be notified of new github teams that can be considered maintainers of a repository. It allows us to exclude teams like `all` from our monitoring and ensure somebody is responsible.